apps/annotations/models.py

```diff
@@ -106,11 +106,24 @@ def system_tags_names(self):
     def all_tag_names(self):
         return [tag["name"] for tag in self.tags_json]
 
+    @cached_property
+    def content_type(self) -> ContentType:
```

Comment:
what's this for?

Reply:
I thought I removed this. I used it before but realized it's not necessary